src/chart/chart.js

```diff
@@ -15,22 +17,27 @@ class Chart {
     }
 
     addData(newData) {
+        newData.forEach(v =>
+            Object.entries(v.values).forEach(([name, value]) => {
+                const side = this.dataSeries[name].yAxisSide;
+                const axis = side === AxisSide.LEFT ? this.metadata.leftAxis : this.metadata.rightAxis;
```

Comment:
I propose splitting this line due to its length.
The format is:
<condition>
? <variant A>
: <variant B>;

Reply:
condidtion
? variantA
: variantB;

Reply:
That's what ternary operator is for - to inline if statements